scripts/templates/error.md

```diff
@@ -1,6 +1,6 @@
 ---
 original: "Original error message"
-excerpt: "Simplified version of the error message (Use "you" as the pronoun because it feels friendly)"
+excerpt: "Simplified version of the error message (Use "you" as the pronoun because it feels friendly and talk as if you were the compiler)"
```

Comment:
```suggestion
excerpt: "Simplified version of the error message"
```

Don't need to add this here because it's already in the contribution instructions.

Reply:
right!